pyproject.toml

```diff
@@ -42,6 +42,9 @@ awscli = [
 boto3 = [
     "boto3 >=1.34.70, <1.34.163",
 ]
+docs = [
```

Comment:
Please, don't abuse extras as dependency groups. Why do you want to provide the end-users with a `pip install 'aiobotocore [docs]'`? It's a public API that you'll end up having to maintain. Besides, putting pinned deps into the installed distribution metadata is a bad tone for libs. Additionally, no dependency management tools (like `pip-tools`) would be able to sanely manage environment pins.

Reply:
Good point! I reverted that part and managed to install the project itself all the same. Looks much better to me now.